js/pillbox.js

```diff
@@ -35,7 +37,15 @@
 
 	// PILLBOX CONSTRUCTOR AND PROTOTYPE
 
-	var Pillbox = function (element, options) {
+	var ENTER_KEYCODE = 13;
+	var COMMA_KEYCODE = 188;// `,` & `<` unless you check for shift modifier key not being pressed on event object (we don't)
+	var isBackspaceKey = function isBackspaceKey(e) { return e.keyCode === 8; };
+	var isDeleteKey = function isDeleteKey(e) { return e.keyCode === 46; };
+	var isTabKey = function isTabKey(e) { return e.keyCode === 9; };
+	var isUpArrow = function isUpArrow(e) { return e.keyCode === 38; };
+	var isDownArrow = function isDownArrow(e) { return e.keyCode === 40; };
```

Comment:
not saying this code refactor isn't good just curious as to why you chose this direction.